src/main/resources/application-jrd-router.yaml

```diff
@@ -14,7 +14,7 @@ batchjob-name: JudicialDataLoad
 scheduler-audit-select: select count(*) from dataload_schedular_audit where date(scheduler_start_time) = current_date
 invalid-exception-sql: insert into dataload_exception_records(file_name,scheduler_start_time,scheduler_name,error_description,updated_timestamp) values(?, ?, ?, ?, ?)
 invalid-jsr-sql: insert into dataload_exception_records(table_name,scheduler_start_time,scheduler_name,key,field_in_error,error_description,updated_timestamp) values(?, ?, ?, ?, ?, ?, ?)
-jsr-threshold-limit: 100
+#jsr-threshold-limit: 100
```

Comment:
can we remove this ?

Reply:
we have explicit kept we want it in JRD....but for some days we want monitor prod without this flag.